kola/tests/kubeadm/kubeadm.go

```diff
@@ -54,8 +54,12 @@ var (
 					_ = c.MustSSH(controller, "/opt/bin/cilium uninstall")
 					version := params["CiliumVersion"].(string)
 					cidr := params["PodSubnet"].(string)
-					cmd := fmt.Sprintf("/opt/bin/cilium install --config enable-endpoint-routes=true --config cluster-pool-ipv4-cidr=%s --version=%s --encryption=ipsec --wait --wait-duration 1m", cidr, version)
-					_ = c.MustSSH(controller, cmd)
+					cmd := fmt.Sprintf("/opt/bin/cilium install --config enable-endpoint-routes=true --config cluster-pool-ipv4-cidr=%s --version=%s --encryption=ipsec --wait-duration=1s --rollback=false", cidr, version)
```

Comment:
Specifying wait duration is most likely pointless if we are not waiting:

```suggestion
					cmd := fmt.Sprintf("/opt/bin/cilium install --config enable-endpoint-routes=true --config cluster-pool-ipv4-cidr=%s --version=%s --encryption=ipsec --rollback=false", cidr, version)
```

Reply:
It actually waits (`--wait` defaults to `true`) - but it made me think that we can `--wait=false --restart-unmanaged-pods=false` to not wait at all, this is enough for this test case.

Reply:
Cool. If that works then feel free to merge the PR.